spinn_utilities/conf_loader.py

```diff
@@ -64,7 +64,7 @@ def install_cfg_and_IOError(filename, defaults, config_locations):
                 raise ConfigTemplateException(
                     f"No template file found for {defaults}")
             else:
-                logger.error(
+                raise ConfigTemplateException(
```

Comment:
Yes, though now the error message itself is maybe a bit confusing?

Reply:
The only way this error is raised is if a template file has been deleted so as long as we understand the error it is fine.

Reply:
OK